src/app/admintoolbar/admintoolbar.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#" style="color:rgb(236, 160, 60); font-size: 40px">Number</a>
+        <a class="navbar-brand" href="#" style="color:rgb(230, 189, 101); font-size: 25px; padding: 9px">Ninja</a>
+      </div>
+      <ul class="nav navbar-nav">
+        <li><a href="#" style="color:yellow">PENDING ACCOUNTS</a></li>
+        <li><a href="#" style="color:yellow">EXISTING ACCOUNTS</a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#"><span class="glyphicon glyphicon-user" style="color:cyan"></span> Log Out</a></li>
+      <!---- <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>-->
```

Comment:
Remove comments.

Reply:
I included this to test the logout button. It has been removed now.